RetailSuiteAccounting/UserInterface/02a_DebtorsCreditors.md

```diff
@@ -0,0 +1,2394 @@
+# Debtors/creditors
+
+*Accounting > Settings > Tab DEBTORS/CREDITORS*
+
+![Customer/Supplier](../../Assets/Screenshots/RetailSuiteAccounting/Settings/CustomerSupplier/CustomerSupplierList.png "[Customer/supplier]")
+
+The *Debtors/creditors* tab displays a customer/supplier list and allows to create, edit and delete customers (debtors) and suppliers (creditors). For detailed information about managing customers and suppliers, see [Manage the customer/supplier](../Integration/05_ManageCustomerSupplier.md).
+
+This tab and all corresponding sub-tabs are also contained in the *Order management* module.
```

Comment:
```suggestion
The identical tab with corresponding sub-tabs is available in the *Order management* module: *Order management > Customers/suppliers*. The customer/supplier list is synchronized between the two modules so that the same data can be accessed.
```